OCS/deploy-ocs.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+export namespace="openshift-storage"
+export rook_override="${rook_override:-false}"
+export public_network="${public_network:-1.1.1.1/24}"
+export cluster_network="${cluster_network:-1.1.1.1/24}"
+
+export rook_cluster=${namespace}
+
+oc patch featureGate cluster --type merge -p '{"spec":{"featureSet":"TechPreviewNoUpgrade"}}'
+
+if [ ! -d rook ]; then
+    git clone https://github.com/rook/rook
+fi
+
+pushd .
+cd rook/cluster/examples/kubernetes/ceph
```

Comment:
Please include these manifests directly in this repository rather than pulling them from an upstream repository

Reply:
Actually no ... won't these manifests be in the OCS Operator?

Reply:
part of the operator indeed